journal/vendor-extra/Views/ViewConverter/StopwatchViewConverter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Libero\Views\ViewConverter;
+
+use Libero\Dom\Element;
+use Libero\Dom\Node;
+use Libero\Views\View;
+use Libero\Views\ViewConverter;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+final class StopwatchViewConverter implements ViewConverter
```

Comment:
So details appear in the Symfony Profiler performance tab.

Reply:
The direction of the dependency looks like it could be reversed, e.g. a `ViewConverter` emits events and there is a listener which could start/stop the stopwatch